src/main/java/zoe/Task.java

```diff
@@ -0,0 +1,64 @@
+package zoe;
+
+/**
+ * Superclass for all forms of tasks
+ * Contains some global methods so that all subclasses have access to that method
+ */
+public class Task {
+    protected String description;
+    protected String type;
+    protected boolean isDone;
+
+    public Task() {
+        this.description = "";
+        this.type = "";
+        this.isDone = false;
+    }
+
+    public Task(String description) {
+        this.description = description;
+        this.type = "";
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    /**
+     * Marks a task as done to reflect in zoe
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Unmarks a given task from list in zoe
+     */
+    public void unmark() {
+        this.isDone = false;
+    }
+
+    public String getStatus() {
+        return String.format("[%s][%s] %s", this.type, this.getStatusIcon(), this.description);
+    }
+
+    /**
+     * Records whether a task is done or not for data loading
+     */
+    public int isDoneNumerical() {
```

Comment:
I thought the return type was boolean. Is it better to start the name with a verb?